modules/custom/openy_group_schedules/src/Form/GroupexFormFull.php

```diff
@@ -587,7 +588,7 @@ public function buildResults(array &$form, FormStateInterface $form_state) {
     }
 
     // Remove instructor parameter in case of no instructor selected.
-    if ($parameters['instructor'] == 'any') {
+    if (isset($parameters['instructor']) && $parameters['instructor'] == 'any') {
```

Comment:
Forgot to mention I added this to fix the php errors that keep showing up with this form.